api/identifiers/serializers.py

```diff
@@ -38,6 +40,11 @@ def get_category(self, obj):
             return 'doi'
         return obj.category
 
+    def get_uri(self, obj):
+        if self.get_category(obj) == 'doi':
+            return f'{settings.DOI_URL_PREFIX}{obj.value}'
```

Comment:
nit: Worth adding a `uri` property to the model? Then this could just be a CharField and the property could also be used in `get_registration_doi_url` below.